modes/gited/evil-collection-gited.el

```diff
@@ -0,0 +1,159 @@
+;;; evil-collection-gited.el --- Evil bindings for Gited -*- lexical-binding: t -*-
+
+;; Copyright (C) 2017 James Nguyen
+
+;; Author: James Nguyen <james@jojojames.com>
+;; Maintainer: James Nguyen <james@jojojames.com>
+;; Pierre Neidhardt <mail@ambrevar.xyz>
+;; URL: https://github.com/emacs-evil/evil-collection
+;; Version: 0.0.1
+;; Package-Requires: ((emacs "26.3") (gited "0.6") (evil "1.2"))
+;; Keywords: evil, gited, tools, vc
+
+;; This program is free software; you can redistribute it and/or modify
+;; it under the terms of the GNU General Public License as published by
+;; the Free Software Foundation, either version 3 of the License, or
+;; (at your option) any later version.
+
+;; This program is distributed in the hope that it will be useful,
+;; but WITHOUT ANY WARRANTY; without even the implied warranty of
+;; MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+;; GNU General Public License for more details.
+
+;; You should have received a copy of the GNU General Public License
+;; along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+;;; Commentary:
+;; Evil bindings for Gited.
+
+;;; Code:
+(require 'gited)
+(require 'evil-collection)
+
+(defconst evil-collection-gited-maps '(gited-mode-map))
+
+;;;###autoload
+(defun evil-collection-gited-setup ()
+  "Set up `evil' bindings for `gited'."
+  (evil-collection-define-key 'normal 'gited-mode-map
+    "q" #'quit-window
+    "g?" #'gited-summary
+    "(" #'gited-hide-details-mode
+    "gr" #'gited-update
+    (kbd "C-c C-c") #'gited-commit
+    (kbd "C-c C-g") #'gited-list-branches
+
+    ;; Movement
+    "j" #'gited-next-line
+    "k" #'gited-prev-line
+    (kbd "M-}") #'gited-prev-marked-branch
+    (kbd "M-}") #'gited-next-marked-branch
+    "G"  #'gited-goto-last-branch
+    "gg" #'gited-goto-first-branch
+    "gj" #'gited-next-marked-branch
+    "gk" #'gited-prev-marked-branch
+    "gl" #'tabulated-list-next-column
+    "gh" #'tabulated-list-previous-column
+    "]]" #'gited-next-marked-branch
+    "[[" #'gited-prev-marked-branch
+    "<"  #'gited-prev-marked-branch
+    ">"  #'gited-next-marked-branch
+    " "  #'gited-next-line
+    [?\S-\ ] #'gited-prev-line
+    [remap next-line] #'gited-next-line
+    [remap previous-line] #'gited-prev-line
+    "J" #'gited-goto-branch
+
+    "gA" #'gited-move-to-end-of-author
+    "ga" #'gited-move-to-author
+    "gB" #'gited-move-to-end-of-branchname
+    "gb" #'gited-move-to-branchname
+    "gD" #'gited-move-to-end-of-date
+    "gd" #'gited-move-to-date
+
+    ;; Operations on Things at Point
+    "{" #'tabulated-list-narrow-current-column
+    "}" #'tabulated-list-widen-current-column
```

Comment:
Those two have been set in ` evil-collection-tabulated-list.el` and can be safely removed. do double check later by `describe-keymap tabulated-list-mode-map`